packages/noir-ethereum-api/src/ethereum/verifier.ts

```diff
@@ -0,0 +1,22 @@
+import ultraVerifier from '../../../../contracts/out/UltraVerifier.sol/UltraVerifier.json';
+import { decodeHexString } from '../noir/noir_js/encode.js';
+import { Account } from 'viem/accounts';
+import { Hex, WalletClient } from 'viem';
+import { WitnessMap } from '@noir-lang/noirc_abi';
+
+export async function verifyStorageProofInSolidity(
+  client: WalletClient,
+  account: Account,
+  contractAddress: Hex,
```

Comment:
I think there is type Address for addresses, instead of Hex.

Reply:
fixed:
`contractAddress: Hex` -> `contractAddress: Address`
return value `Promise<Hex>` -> `Promise<Hash>`